modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -2,7 +2,7 @@
 /**
 * The user account page
 *
-* PHP Version 5
+* PHP Version 7
```

Comment:
The PHP doc specifies what the minimum version of PHP required for the code in that file to run is, not the version of PHP that LORIS supports.